src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
     shouldShowOnfido: PropTypes.bool.isRequired,
 };
 
-class RequestorStep extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.validate = this.validate.bind(this);
-        this.submit = this.submit.bind(this);
-    }
-
-    /**
-     * @param {Object} values
-     * @returns {Object}
-     */
-    validate(values) {
-        const requiredFields = ['firstName', 'lastName', 'dob', 'ssnLast4', 'requestorAddressStreet', 'requestorAddressCity', 'requestorAddressState', 'requestorAddressZipCode'];
-        const errors = ValidationUtils.getFieldRequiredErrors(values, requiredFields);
-
-        if (values.dob) {
-            if (!ValidationUtils.isValidPastDate(values.dob) || !ValidationUtils.meetsMaximumAgeRequirement(values.dob)) {
-                errors.dob = 'bankAccount.error.dob';
-            } else if (!ValidationUtils.meetsMinimumAgeRequirement(values.dob)) {
-                errors.dob = 'bankAccount.error.age';
-            }
-        }
-
-        if (values.ssnLast4 && !ValidationUtils.isValidSSNLastFour(values.ssnLast4)) {
-            errors.ssnLast4 = 'bankAccount.error.ssnLast4';
-        }
+const REQUIRED_FIELDS = ['firstName', 'lastName', 'dob', 'ssnLast4', 'requestorAddressStreet', 'requestorAddressCity', 'requestorAddressState', 'requestorAddressZipCode'];
+const INPUT_KEYS = {
+    firstName: 'firstName',
+    lastName: 'lastName',
+    dob: 'dob',
+    ssnLast4: 'ssnLast4',
+    street: 'requestorAddressStreet',
+    city: 'requestorAddressCity',
+    state: 'requestorAddressState',
+    zipCode: 'requestorAddressZipCode',
+};
+const STEP_COUNTER = {step: 3, total: 5};
+
+const STYLES = {
```

Comment:
I don't see the value of adding the styles into an object, also isn't the standard in the codebase. I will keep the styles directly into the components

Reply:
Done ✅